go/kbfs/libkbfs/disk_block_cache.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultDiskBlockCacheMaxBytes   uint64 = 10 * (1 << 30)
 	defaultBlockCacheTableSize      int    = 50 * opt.MiB
 	defaultBlockCacheBlockSize      int    = 4 * opt.MiB
+	defaultBlockCacheCapacity       int    = 8 * opt.MiB
```

Comment:
Is this best to only be double the block size?

Reply:
It was previously 4MB - I tried larger values and they empirically didn't give better results (equal performance [within measurement error] and with more resource usage if I understand that parameter correctly).